tests/e2e/cypress/cypress/support/navigation.js

```diff
@@ -50,17 +49,19 @@ Cypress.Commands.add("goToMenu", (menuId) => {
   cy.get('#' + menuId).click()
 })
 
-Cypress.Commands.add("goToItem", (itemBarcode) => {
+Cypress.Commands.add("goToDocumentDetailView", (itemBarcode) => {
   // Go to homepage
   cy.get('#homepage-logo').click()
   cy.wait(800)
   cy.url().then((url) => {
     if (url.includes('/professional/')) {
       // on professional context
+      cy.get('.form-control').clear()
       cy.get('.form-control').type(itemBarcode).type('{enter}')
     }
     else {
        // on public context
+      cy.get('.d-none > main-search-bar > .flex-grow-1 > .rero-ils-autocomplete > .form-control').clear()
```

Comment:
Could be simplify `main-search-bar input[@name="q"]` ?

Reply:
Yes, asap